packages/embeds/embed-core/src/embed.ts

```diff
@@ -711,3 +750,18 @@ document.addEventListener("click", (e) => {
     calOrigin,
   });
 });
+
+let currentColorScheme: string | null = null;
+
+(function watchAndActOnColorSchemeChange() {
+  setInterval(() => {
+    const colorScheme = getColorScheme(document.body);
+    if (colorScheme && colorScheme !== currentColorScheme) {
```

Comment:
Avoid bombarding iframe with colorScheme setting instructions. Only do it when it actually changes.